README.md

```diff
@@ -1,11 +1,12 @@
-The irresponsible clojure guild present...
+The irresponsible clojure guild presents...
 
 # domiscuity
 
-Parse html5 and play with the DOM with a clean API
+[![Build Status](https://travis-ci.org/kalouantonis/domiscuity.svg?branch=master)](https://travis-ci.org/kalouantonis/domiscuity)
```

Comment:
this should probably say "irresponsible" not "kaloartugfradrefsahtgugadebtygyautsimdyslexica"